### ISSUE TITLE
[Livestream] When Starting / Stopping Livestream - Unhandled Rejection Thrown

### ISSUE THREAD

**Issue by thorbenprimke** [daily-co/react-native-daily-js#28]
```
 LOG  Starting live stream to: rtmps://redacted/
 WARN  Possible Unhandled Promise Rejection (id: 2):
ReferenceError: Can't find variable: status
@
l@
@
r@
s@
tryCallOne@http://192.168.6.22:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=org.reactjs.native.example.DailyPlayground:28451:16
```

The above error is thrown both on starting/stopping a livestream. I don't think it's impacting the actual functionality but it's annoying. [The above log is from the Daily React Native Playground app with the live streaming branch rebased onto latest](https://github.com/thorbenprimke/react-native-daily-js-playground/tree/demo-live-streaming).

# Expected behavior

No rejection being thrown / being unhandled. 

# Describe the bug (unexpected behavior)

That's all the logs I see. I searched both daily-js and react-native-daily-js repos but didn't find a reference to `stream` so it must be in code that's pulled in dynamically.

# Steps to reproduce

1. Join a room
2. Start the live stream

# System information 

* Device: iPhone X
* OS, version: 15.5
* App: Daily RN Playground
* Dependencies:
```
    "@daily-co/react-native-daily-js": "^0.25.0",
    "@daily-co/react-native-webrtc": "1.94.1-daily.4",
```

**Comment by filipi87** [comment]
Hi @thorbenprimke , thanks a lot for reporting that. I have already been able to reproduce the issue here. I will let you know as soon as we fix It.

**Comment by filipi87** [comment]
Hi, just to give you an update, the fix has already been merged and will be available on the next release, 0.26.0.
